.github/workflows/dockerhub-latest-image.yml

```diff
@@ -0,0 +1,123 @@
+name: Push latest image to DockerHub
+
+on:
+  push:
+    branches:
+      - main
+
```

Comment:
Please declare permissions here, like: https://github.com/karmada-io/karmada/blame/master/.github/workflows/dockerhub-latest-image.yml#L6-L7.

Reply:
thanks for pointing that, the limited permission has added ~